src/main/java/seedu/fridgefriend/utilities/Parser.java

```diff
@@ -113,15 +118,28 @@ public static Command parseCommand(String[] parsedInput)
     }
 
     /**
-     * Define arguments format for add food command.
+     * Define arguments format for add food command with quantity.
      * A Pattern object which defines how the input string for food item
      * that should look like. [^/]+ implies 1 or more characters except for '/'
      */
-    public static final Pattern FOOD_DATA_ARGS_FORMAT =
+    public static final Pattern FOOD_DATA_ARGS_FORMAT_QUANTITY =
             Pattern.compile("(?<name>[^/]+)"
                     + " /cat (?<category>[^/]+)"
                     + " /exp (?<expiryDate>[^/]+)"
-                    + " /loc (?<storageLocation>[^/]+)");
+                    + " /loc (?<storageLocation>[^/]+)"
+                    + " /qty (?<quantity>[^/]+)");
+
+    /**
+     * Define arguments format for add food command with weight.
+     * A Pattern object which defines how the input string for food item
+     * that should look like. [^/]+ implies 1 or more characters except for '/'
+     */
+    public static final Pattern FOOD_DATA_ARGS_FORMAT_WEIGHT =
```

Comment:
Do you plan to add a pattern for every child class of QUANTITY? But current implementation can work.

Reply:
No. I was thinking that these two would be sufficient as we would have the user enter the amount either by /qty or /wgt only